assets/custom-semantic-ui/dist/semantic.css

```diff
@@ -40205,6 +40205,27 @@ body.pushable > .pusher {
   }
 }
 
+
```

Comment:
@kadevgraaf are we allowed to change css like this? Don't we have a separate repo we update all this css from ?

Reply:
We have our own version of Semantic UI, so it is fine to change, but @abijames  might want to take a look at the changes.

Reply:
talking about future issues with this repo: https://github.com/slidewiki/semantic-ui-customized from what i understand we prepare changes there and merge later in platform, like in https://github.com/slidewiki/slidewiki-platform/pull/1059.

Also, @abijames has already approved the UI for this, just doing some code and sanity checking here.

Reply:
@nk1g09 as discussed with @abijames please don't change anything in `assets/custom-semantic-ui/` folders, and move whatever custom classes you have in `assets/css/questions.css` (create the file). then just import that file in all components that make use of it, like: `import 'assets/css/questions.css'` (no need to put in some variable).

Reply:
So I tried this... but when i do npm run dev after the changes i get the error shown below in compilation. It then correctly compiles but I can't access any of the site as I get a 504 error. Have I made some silly error...? My css looks like the other files in the assets/css/ folder. 

SyntaxError: C:/github/slidewiki-platform/assets/css/questions.css: Unexpected token (4:0)
  2 | /* Embed Questions */
  3 |
> 4 | .ui.questions.padded.text.grey{
    | ^
  5 |     color: #767676 !important;
  6 | }
  7 | .ui.segment.questions.unpadded{
    at Parser.pp$5.raise (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:4454:13)
    at Parser.pp.unexpected (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:1761:8)
    at Parser.pp$3.parseExprAtom (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:3750:12)
    at Parser.parseExprAtom (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:7238:22)
    at Parser.pp$3.parseExprSubscripts (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:3494:19)
    at Parser.pp$3.parseMaybeUnary (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:3474:19)
    at Parser.pp$3.parseExprOps (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:3404:19)
    at Parser.pp$3.parseMaybeConditional (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:3381:19)
    at Parser.pp$3.parseMaybeAssign (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:3344:19)
    at Parser.parseMaybeAssign (C:\github\slidewiki-platform\node_modules\babylon\lib\index.js:6474:20)
[nodemon] app crashed - waiting for file changes before starting...

Reply:
It's OK, not sure if my instructions where correct anyways. If you can, please add whatever you have on a separate branch and push it so i can check it out, thanks!

Reply:
https://github.com/slidewiki/slidewiki-platform/tree/Quiz-tool-css 
pushed the changes to here.